Mathlib/MeasureTheory/Constructions/Pi.lean

```diff
@@ -970,6 +970,26 @@ theorem volume_preserving_piFinsetUnion (α : ι → Type*) [DecidableEq ι] {s
     MeasurePreserving (MeasurableEquiv.piFinsetUnion α h) volume volume :=
   measurePreserving_piFinsetUnion h (fun _ ↦ volume)
 
+theorem measurePreserving_pi {β : ι → Type*} [∀ i, MeasurableSpace (β i)]
```

Comment:
Can't you deduce the sigma-finiteness of `mu i` from the one of `nu i` instead of assuming it, thanks to [MeasureTheory.MeasurePreserving.sigmaFinite](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Dynamics/Ergodic/MeasurePreserving.html#MeasureTheory.MeasurePreserving.sigmaFinite)? Same thing for the next lemma.

Reply:
Indeed. Thanks!